aiida/cmdline/params/options/commands/code.py

```diff
@@ -60,25 +66,32 @@ def is_not_on_computer(ctx):
     prompt_fn=is_not_on_computer,
     type=click.Path(dir_okay=False),
     cls=InteractiveOption,
-    help=('[if --store-in-db]: relative path of the executable inside the code-folder.')
+    help=('[if --store-in-db]: Relative path of the executable inside the code-folder.')
 )
 
-LABEL = options.LABEL.clone(prompt='Label', cls=InteractiveOption, help='A label to refer to this code.')
+LABEL = options.LABEL.clone(
+    prompt='Label',
+    cls=InteractiveOption,
+    help='A label to refer to this code.' +
+    '  Uniqueness is not enforced, but is required to identify a code by its label (rather than its PK or UUID).'
```

Comment:
```suggestion
    help="This label can be used to identify the code (using 'label@computername'), as long as labels are unique per computer."
```

Reply:
perhaps better 'label@computerlabel'

seb might nit-pick that you can also use the label without the computer name... but for me it's fine